_faqs/faq-12.md

```diff
@@ -1,8 +1,27 @@
 ---
-question: I want to cite RIOT. Which reference should I use?
+question: How does RIOT compare to XYZ?
```

Comment:
"XYZ" is not nice.
maybe something like:
... to other operating systems for embedded or IoT devices.